app/presenters/openid_connect_certs_presenter.rb

```diff
@@ -1,13 +1,19 @@
 class OpenidConnectCertsPresenter
   def certs
     {
-      keys: keys.map { |key| JWT::JWK.new(key).export },
+      keys: keys
     }
   end
 
   private
 
   def keys
-    [AppArtifacts.store.oidc_public_key]
+    [AppArtifacts.store.oidc_public_key].map do |key|
+      export = JWT::JWK.new(key).export
+      export[:alg] = 'RS256'
+      export[:use] = 'sig'
+
+      export
```

Comment:
six of one, half a dozen of the other, could save a variable and use `.merge`?

```suggestion
      JWT::JWK.new(key).export.merge(
        alg: 'RS256',
        use: 'sig',
      )
```

Reply:
I was hoping to find a `put_value_but_raise_if_the_key_already_exists` in case `export` behavior changes and we were worried about overwriting, but the merge suggestion gave me the idea to reverse the order of your merge?

Reply:
that makes sense!

Reply:
Done in [e82ad45](https://github.com/18F/identity-idp/pull/6007/commits/e82ad45483005222d9eecbad74454161cc0252a5)